sql/moz-fx-data-shared-prod/ltv_state_values_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: LTV State Values Derived
+description: |
+  Dataset where the data science team has write access to write LTV state values from their models
+dataset_base_acl: derived_restricted
+user_facing: false
+labels: {}
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:datascience/developers
```

Comment:
This is redundant with the write access granted and the read access granted to `workgroup:mozilla-confidential` which is a superset of `workgroup:datascience/developers`.
```suggestion
```